frontend/src/styles/README.md

```diff
@@ -0,0 +1,175 @@
+# Relay Styles
+
+This folder contains style definitions used across the Relay dashboard:
+
+- `globals.scss` - Implements a [CSS Reset][], adds some top-level styles.
+- `colors.scss` - Implements the colors for the [Nebula Design System][],
+  overriding the [Mozilla Protocol colors][].
+- `text.scss` - Implements typography mixins that extend [Mozilla Protocol typography][] to take a [content block][] to avoid [mixed declarations][].
+
+[CSS Reset]: https://en.wikipedia.org/wiki/Reset_style_sheet
+[Mozilla Protocol colors]: https://protocol.mozilla.org/docs/fundamentals/color
+[Mozilla Protocol typography]: https://protocol.mozilla.org/docs/fundamentals/typography
+[Nebula Design System]: https://www.peterbenvenuto.com/nebula
+[content block]: https://sass-lang.com/documentation/at-rules/mixin/#content-blocks
+[mixed declarations]: https://sass-lang.com/documentation/breaking-changes/mixed-decls/
+
+## Colors
+
+The [Nebula Design System][] was launched in 2023 to implement a shared style across the Privacy and Security products ([Mozilla VPN][], [Mozilla Monitor][], and Firefox Relay). It defines a consistent typography and color scheme for applications, web pages, and marketing materials.
+
+The system is implemented by defining the colors in `colors.scss`, and adjusting other styles as needed. These colors are used with this pattern:
+
+```scss
+@use "~@mozilla-protocol/core/protocol/css/includes/lib" as *;
+@use "../styles/color";
+
+.warning {
+  background-color: color.$white;
+  border-color: color.$yellow-50;
+}
+```
+
+When using the [Mozilla Protocol colors][], the pattern is:
+
+```scss
+@use "~@mozilla-protocol/core/protocol/css/includes/lib" as *;
+@use "../styles/color";
```

Comment:
question (non-blocking): These Mozilla Protocol `@use` statements are identical to the Nebula ones above? Are they supposed to be different? Or is it really the case that the Nebula color variables are in the `color` namespace and the Protocol colors are `$color-*` variables in the global namespace? I think you mention this in the PR description, but maybe add that explanation to this `README.md` too?

Reply:
yeah, that is not clear to me either. The CSS uses the the actual values (like `color:#ffa436`), and the CSS didn't change :broken_record:.

I think it is a separate task to confirm we used the colors we intended to use, since that may change the CSS, and may break E2E screenshot tests. I've opened [MPP-3958](https://mozilla-hub.atlassian.net/browse/MPP-3958) for that work, I'll add a reference to the README.